src/libcore/convert.rs

```diff
@@ -99,28 +98,20 @@ use fmt;
 #[inline]
 pub const fn identity<T>(x: T) -> T { x }
 
-/// A cheap reference-to-reference conversion. Used to convert a value to a
-/// reference value within generic code.
-///
-/// `AsRef` is very similar to, but serves a slightly different purpose than,
-/// [`Borrow`].
+/// Used to do a cheap reference-to-reference conversion.
```

Comment:
There needs to be a `///` between this line and the next one; the format is

```rust
/// summary line
///
/// full description
```

the previous doc block didn't follow this closely, so it's not your fault! but it's a thing we should fix. :)